test/features/cloudnative/init_containers.go

```diff
@@ -46,18 +52,18 @@ func checkInitContainers(sampleApp *sample.App) features.Func {
 			var oneAgentInstallContainer *corev1.Container
 
 			for _, initContainer := range podItem.Spec.InitContainers {
-				if initContainer.Name == webhook.InstallContainerName {
+				if initContainer.Name == initContainerName {
```

Comment:
just add an `||` and have
```go
const oldInstallContainerName = "install-oneagent"
...

if initContainer.Name == webhook.InstallContainerName || initContainer.Name == oldInstallContainerName
```

or you can have `slices.Contains` and have a slice of possible init-container names 😅 
```go
var initNames = []string{...}
...
if slices.Contains(initNames, initContainer.Name)
```

Reply:
okey will do.

Reply:
fixed in abb030ced